test/asb-helper.js

```diff
@@ -0,0 +1,57 @@
+const { ReceiveMode, ServiceBusClient } = require('@azure/service-bus')
+const config = require('../server/config/mq-config')
+
+// When calling this witin a test script, ensure there is a generous timeout
+// for the connections to complete within. `30000` should be good
```

Comment:
If we're awaiting the callback why do we need to wait a further 30 seconds?  Are there connection closure activities that take place outside of the code we need to wait to complete?  If we don't what is the impact on the tests?

Just concerned that if we add 30 seconds to every test suite then our test run could really slow down

Reply:
Jest will only wait 5 seconds before it considers there to be a problem. The 30 seconds isn't a pure wait, as such, just an amount of time that it could wait, if needed.
The change of `ReceiveMode` to `receiveAndDelete` makes quite a difference to the time take, at least IME.